### DISCUSSION TITLE
Curve Storage Orchestration for Kubernetes

### DISCUSSION THREAD

**Discussion by zhanghuidinah** [opencurve/curveadm#140] [Ideas]
We needs turns distributed storage systems into self-managing, self-scaling, self-healing storage services. It automates the tasks of a storage administrator: deployment, bootstrapping, configuration, provisioning, scaling, upgrading, migration, disaster recovery, monitoring, and resource management.

We can uses the power of the Kubernetes platform to deliver its services via a **Kubernetes Operator for Curve**.

**Comment by zhanghuidinah** [comment]
FYI https://github.com/rook/rook

**Comment by zhanghuidinah** [comment]
We need a meaningful name for this project.

**Reply by zhuyaguang** [reply]
curve-operator

**Reply by Wine93** [reply]
agree!

**Reply by logn-xu** [reply]
> curve-gourd 哈哈哈 我不知道为啥想到了 葫芦 gourd

**Reply by WyattJia** [reply]
叫 angle ，和 curve 一样都是几何系的名字。

**Reply by logn-xu** [reply]
hhh  圆锥曲线的感觉
